### ISSUE TITLE
evaluation by group in documentation should be evaluation_by_group

### ISSUE THREAD

**Issue by bndgyawali** [EducationalTestingService/rsmtool#102]
In RSMTool as well as RSMCompare documentation, it should be `evaluation_by_group`
Ref: http://rsmtool.readthedocs.io/en/latest/usage_rsmtool.html#general-sections-optional

**Comment by bndgyawali** [comment]
Seems like `evaluation_by_group` is not available in `RSMCompare` but is present in the documentation.

```
Traceback (most recent call last):
  File "/opt/python/rsmtool/bin/rsmcompare", line 11, in <module>
    sys.exit(main())
  File "/opt/python/rsmtool/lib/python3.4/site-packages/rsmtool/rsmcompare.py", line 220, in main
    run_comparison(config_file, output_dir)
  File "/opt/python/rsmtool/lib/python3.4/site-packages/rsmtool/rsmcompare.py", line 160, in run_comparison
    context='rsmcompare')
  File "/opt/python/rsmtool/lib/python3.4/site-packages/rsmtool/report.py", line 438, in get_ordered_notebook_files
    context=context)
  File "/opt/python/rsmtool/lib/python3.4/site-packages/rsmtool/report.py", line 286, in determine_chosen_sections
    check_section_names(general_sections, 'general', context)
  File "/opt/python/rsmtool/lib/python3.4/site-packages/rsmtool/report.py", line 191, in check_section_names
    master_section_list))
ValueError: The following general report section names are invalid or not supported for rsmcompare: {'evaluation_by_group'}
The following sections are currently available: ['feature_descriptives', 'features_by_group', 'preprocessed_features', 'preprocessed_features_by_group', 'consistency', 'score_distributions', 'model', 'evaluation', 'pca', 'notes', 'sysinfo']
```

**Comment by desilinguist** [comment]
Addressed by #106
